dvmkv2mp4

```diff
@@ -303,7 +304,9 @@ EOF
 
 
   ### GRAB CHAPTERS
-  mkvextract chapters -s "${output}.asm" > chapters.list
+  if [ -s "${output}.asm" ]; then
```

Comment:
Tbh i would completely change that and instead during extract from asm do extract from input mkv. That would solve issues when someone does not use  -a switch